sw/device/tests/dif/dif_otbn_smoketest.c

```diff
@@ -56,6 +56,17 @@ static void check_otbn_err_bits(otbn_t *otbn_ctx,
         otbn_err_bits);
 }
 
+/**
+ * Get OTBN instruction count, check that it matches `expected_insn_cnt`.
+ */
+static void check_otbn_insn_cnt(otbn_t *otbn_ctx, uint32_t expected_insn_cnt) {
+  uint32_t insn_cnt;
+  CHECK(dif_otbn_get_insn_cnt(&otbn_ctx->dif, &insn_cnt) == kDifOtbnOk);
+  CHECK(insn_cnt == expected_insn_cnt,
+        "dif_otbn_insn_cnt() produced unexpected instruction count: %d",
```

Comment:
The function doesn't really "produce" the instruction count, how about something like this?

```suggestion
        "Expected to execute %d instructions, but got %d.",
```